ui/src/project.ts

```diff
@@ -104,8 +113,20 @@ interface FetchUser extends event.Event<Kind> {
   urn: string;
 }
 
-type Msg = Create | Fetch | FetchList | FetchUser;
+interface FetchLocalState extends event.Event<Kind> {
+  kind: Kind.FetchLocalState;
+  path: string;
+}
+
+type Msg =
+  | ClearLocalState
+  | Create
+  | Fetch
+  | FetchList
+  | FetchLocalState
+  | FetchUser;
 
+// / REQUEST INPUTS
```

Comment:
Is this a typo or some special type of comment in ts? Couldn't find anything on it...

```suggestion
// REQUEST INPUTS
```

Reply:
Just a typo 🤡